subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan.c

```diff
@@ -109,7 +110,8 @@ void lll_scan_prepare(void *param)
 	err = lll_hfclock_on();
 	LL_ASSERT(err >= 0);
 
-	err = lll_prepare(is_abort_cb, abort_cb, prepare_cb, 0, param);
+	/* Process prepare with a priority of +1 */
+	err = lll_prepare(is_abort_cb, abort_cb, prepare_cb, 1, param);
```

Comment:
Add comments on why the priority of +1.

Reply:
The LL scheduler is currently oblivious of "age" priorities and with default priority ADV would "loose" to scan, in e.g. the adv_scan test, every other time. By setting scan to +1, ADV wins that case every time and, normal advertising presence is obtained. This could also be achieved by means of is_abort, by creating some interaction rules. As first shot I felt this would do - also to show how prio can be used.